assets/javascripts/audit_log.js

```diff
@@ -200,7 +200,7 @@ function rescheduleProduct(url) {
       const id = jqXHR.responseJSON?.scheduled_product_id;
       const msg =
         typeof id === 'number'
-          ? `The product has been re-triggered as <a href="/admin/productlog?id=${id}">${id}</a>.`
+          ? 'The product has been re-triggered as <a href="/admin/productlog?id=${id}">${id}</a>.'
```

Comment:
But then the string interpolation won't work anymore, so you have to concatenate the id manually I guess
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
I amended the commit